bundle/edu.gemini.pot/src/main/scala/edu/gemini/pot/ModelConverters.scala

```diff
@@ -26,6 +26,25 @@ object ModelConverters {
     def toOldModel: skycalc.Angle = skycalc.Angle.degrees(angle.toDegrees)
   }
 
+  // We need a way to convert angles from [0,maxValOfUnit) to [-maxValOfUnit/2,maxValOfUnit/2) for a number of purposes.
+  private val maxRadians = 2 * math.Pi
+  private val maxDegrees = 360
+  private val maxArcmins = 60 * maxDegrees
+  private val maxArcsecs = 60 * maxArcmins
+  implicit class AngleNormalizer(val angle: Angle) extends AnyVal {
+    private def normalize(value: Double, maxValue: Double): Double = {
+      val half = maxValue / 2.0
+      val pos = value % maxValue
+      val neg = pos - maxValue
+      if (math.abs(pos) < math.abs(neg)) pos else neg
+    }
```

Comment:
I've been hesitant to fix the offsetting issue with this, because it seems to happen in a few places (e.g. in multiple places throughout the `PatrolField` code), and I'm not sure where all of them are of if this is the solution that we want to ultimately use, so I've held off. I think for now, I'll tell Andy and Bryan to stick to positive offsets.

This was needed for the TPE for now, though, because if either coordinate of the base position ever becomes negative, the science area ends up being drawn in the wrong place because of the positive angle restriction.

I'll go ahead and make the changes as you recommended and then merge unless someone speaks up now and says otherwise :-).